implementation/configuration/src/configuration_impl.cpp

```diff
@@ -41,6 +41,8 @@
 #include "../../security/include/policy_manager_impl.hpp"
 #include "../../security/include/security.hpp"
 
+#include <ifaddrs.h>
```

Comment:
We cannot accept a PR that breaks our windows build, this is linux only.

Reply:
added `if defined(__linux__) .... #endif` to address it